src/components/Form/FieldUpload/FieldUploadPreview.tsx

```diff
@@ -0,0 +1,108 @@
+import { FC, PropsWithChildren, useCallback, useEffect, useState } from 'react';
+
+import { Box, Flex, type FlexProps, IconButton } from '@chakra-ui/react';
+import { useFormContext } from 'react-hook-form';
+import { LuX } from 'react-icons/lu';
+
+const ImagePreview = ({
+  image,
+  onClick,
+}: {
+  image: string;
+  onClick: React.MouseEventHandler<HTMLButtonElement>;
+}) => {
+  return (
+    <Box
+      position="relative"
+      bg="gray.200"
+      height="100%"
+      aspectRatio="1"
+      overflow="hidden"
+      rounded="md"
+    >
+      <Box as="img" width="100%" height="100%" objectFit="cover" src={image} />
+      <IconButton
+        position="absolute"
+        top="1"
+        right="1"
+        icon={<LuX fontSize="sm" />}
+        aria-label="Remove"
+        rounded="full"
+        minWidth="6"
+        minHeight="6"
+        width="6"
+        height="6"
```

Comment:
```suggestion
        width="6"
        height="6"
```

I think it is enough like this :)